packages/app/src/store/aut-store.ts

```diff
@@ -76,6 +76,10 @@ export const useAutStore = defineStore({
       this.isLoadingUrl = false
     },
     setScriptError (err: ScriptError) {
+      if (typeof err === 'string') {
```

Comment:
Was wondering the same thing here, it looks like 
```ts
export type ScriptError = { type: string, error: string } | null
``` 
so a string check here would be unexpected. It would be good to either update the type or cast to the proper type properly wherever this is happening?

Reply:
This was but then I fixed the cra-handler. I need to dig in again to see how this error originated as a string and not `ScriptError`. I'll revert this change and create a new issue for what I find.

Reply:
[37155e9](https://github.com/cypress-io/cypress/pull/20954/commits/37155e9ded58a3dfa452427c44a0785c8feff2b5) couldn't reproduce so reverted.